### ISSUE TITLE
Connection close on 0.10.x branch

### ISSUE THREAD

**Issue by jfrey** [balderdashy/sails-mongo#321]
Hi All,

I noticed this https://github.com/balderdashy/sails-mongo/commit/30034f3d927920d9c5ac6d9747fe28581cbda1b1 commit in master/0.11.x and wanted to see if we could cherry pick it into the 0.10.x branch for <=2.4.5 Mongo versions?

I can setup a PR for it if that would be helpful.

Thanks!

**Comment by devinivy** [comment]
Hi @jfrey– very sorry for the slow response.  I think that would be nice!  @particlebanana mind publishing another patch to 0.10.x?

**Comment by sailsbot** [comment]
Thanks for posting, @jfrey.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by tldavies** [comment]
:+1:

**Comment by sailsbot** [comment]
Thanks for posting, @jfrey.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by devinivy** [comment]
@particlebanana ping!  How do you feel about this?

**Comment by particlebanana** [comment]
On it

**Comment by particlebanana** [comment]
published `0.10.9`
